.evg.yml

```diff
@@ -384,3 +395,25 @@ functions:
         working_dir: mongo-jdbc-driver
         script: |
           PROJECT_DIRECTORY="$(pwd)" NEXUS_PROFILE_ID=${NEXUS_PROFILE_ID} NEXUS_USERNAME=${NEXUS_USERNAME} NEXUS_PASSWORD=${NEXUS_PASSWORD} NEXUS_URL=${NEXUS_URL} SIGNING_PASSWORD=${SIGNING_PASSWORD} SIGNING_KEY_ID=${SIGNING_KEY_ID} RING_FILE_GPG_BASE64=${RING_FILE_GPG_BASE64} ./gradle/publish.sh
+
+  "update download center feed":
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: mongo-jdbc-driver
+        script: |
+          ${prepare_shell}
+          echo "Version = ${MDBJDBC_VER}"
+          sed -i "s@{RELEASE_VERSION}@${MDBJDBC_VER}@" mongo-jdbc-downloads_template.json
+          echo "-------------------------"
+          cat mongo-jdbc-downloads_template.json
+          echo "-------------------------"
+    - command: s3.put
+      params:
+        aws_key: ${release_aws_key}
```

Comment:
These keys will need to be added into the evergreen project.

Reply:
ah, very good point!

Reply:
Done! Thank you for flagging this and avoid us loosing time figuring this out at the next release.